.github/workflows/coverage.yml

```diff
@@ -0,0 +1,41 @@
+name: calculate coverage
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  calc_coverage:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Setup conda
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          miniconda-version: 'latest'
+          python-version: 3.7
+          activate-environment: test
+          environment-file: .github/test_conda_env.yml
+          condarc-file: .github/test_condarc.yml
+
+      - name: install obspy
+        shell: bash -l {0}
+        run: |
+          pip install -e .
+
+      - name: run test suite
+        shell: bash -l {0}
+        run: |
+          obspy-runtests
```

Comment:
there's more to add here, but I'm pretty sure you're aware and just wanted to keep it simple for the first tests :+1: 

```
  - if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
      coverage run --rcfile=.coveragerc --source=${MODULELIST} -m obspy.scripts.runtests --no-flake8 --all -n travis-ci -r --ci-url="https://travis-ci.org/${TRAVIS_REPO_SLUG}/jobs/${TRAVIS_JOB_ID}";
    else
      coverage run --rcfile=.coveragerc --source=${MODULELIST} -m obspy.scripts.runtests --no-flake8 -n travis-ci -r --ci-url="https://travis-ci.org/${TRAVIS_REPO_SLUG}/jobs/${TRAVIS_JOB_ID}" --pr-url="https://github.com/obspy/obspy/pull/${TRAVIS_PULL_REQUEST}" $MODULELISTSPACES;
```

Reply:
Yes sir. Hopefully GHA gives us the the PR URL, need to look into it more.